sync.cmd

```diff
@@ -0,0 +1,25 @@
+@if "%_echo%" neq "on" echo off
+setlocal
+
+set _args=%*
+if /I [%1]==[/?] goto Usage
+if [%1]==[]  set _args=-p
+
+echo Running executor.cmd %_args%
+call executor.cmd %_args%
```

Comment:
By default it is passing -p that is the same thing as sync, I'll change it to `executor.cmd sync %*` as it is a little confusing.

In the corefx repo sync has 3 options: 
-p (restore packages)
-t (creates project.json and restore those packages for testing against packages)
-ab (downloads the product packages stored in Azure)

using the executor will look like:
```
executor.cmd sync                                  == executor -p
executor.cmd testAgainstPackages        == executor.cmd -t
executor.cmd downloadAzurePackages == executor.cmd -ab
```
@Chrisboh mentioned that it would be better if we have something more like 
```
executor.cmd sync -p
executor.cmd sync -ab
```
This last part would involve more changes to the executor but it may help to keep things more organized.
Is there a preference?